File: integration-tests/mongodb-panache/src/main/java/io/quarkus/it/mongodb/panache/reactive/book/ReactiveBookRepository.java
package io.quarkus.it.mongodb.panache.reactive.book;

import jakarta.enterprise.context.ApplicationScoped;

import io.quarkus.it.mongodb.panache.book.Book;
import io.quarkus.mongodb.panache.reactive.ReactivePanacheMongoRepository;

@ApplicationScoped
public class ReactiveBookRepository implements ReactivePanacheMongoRepository<Book> {
}

File: integration-tests/mongodb-panache-kotlin/src/main/kotlin/io/quarkus/it/mongodb/panache/reactive/book/ReactiveBookRepository.kt
package io.quarkus.it.mongodb.panache.reactive.book

import io.quarkus.it.mongodb.panache.book.Book
import io.quarkus.mongodb.panache.kotlin.reactive.ReactivePanacheMongoRepository
import jakarta.enterprise.context.ApplicationScoped

@ApplicationScoped
class ReactiveBookRepository : ReactivePanacheMongoRepository<Book>
